dist/rpm/openQA.spec

```diff
@@ -563,8 +563,13 @@ fi
 %{_sbindir}/rcopenqa-webui
 %{_sbindir}/rcopenqa-livehandler
 %dir %{_sysconfdir}/openqa
-%config(noreplace) %attr(-,geekotest,root) %{_sysconfdir}/openqa/openqa.ini
-%config(noreplace) %attr(-,geekotest,root) %{_sysconfdir}/openqa/database.ini
+%dir %{_sysconfdir}/openqa/openqa.ini.d
+%dir %{_sysconfdir}/openqa/database.ini.d
+%dir %{_distconfdir}/openqa
+%dir %{_distconfdir}/openqa/openqa.ini.d
+%dir %{_distconfdir}/openqa/database.ini.d
+%config(noreplace) %attr(-,geekotest,root) %{_datadir}/doc/openqa/examples/openqa.ini
```

Comment:
Why `noreplace` for the example configs? I guess we would never edit them

Reply:
Right, this is a leftover I need to get rid of.